pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -17,102 +17,107 @@
-                                    "<%= tag.name %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= tag.name %>\b/.test(e);
-                                    },
+                    <div class="table-responsive">
+                        <table id="questionsTable" class="table table-sm table-hover tablesorter">
+                            <thead>
+                                <tr>
+                                    <th>Title</th>
+                                    <th>QID</th>
+                                    <th class="filter-select filter-exact" data-placeholder="Choose topic">Topic</th>
+                                    <th data-placeholder="Choose tag">Tags</th>
+                                    <% if (locals.course_instance) { %>
+                                        <th data-placeholder="Choose assessment">Assessments</th>
+                                        <% } %>
+                                            <th class="filter-select filter-exact" data-placeholder="Choose Version">Version</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <% questions.forEach(function(question) { %>
+                                    <tr>
+                                        <td class="align-middle">
+                                            <a href="<%= urlPrefix %>/question/<%= question.id %>/">
+                                                <%= question.title %>
+                                            </a>
+                                            <%- include('../partials/issueBadge', {count: question.open_issue_count, issueQid: question.qid}); %>
+                                        </td>
+                                        <td class="align-middle">
+                                            <%= question.qid %>
+                                        </td>
+                                        <td class="align-middle">
+                                            <%- include('../partials/topic', {topic: question.topic}); %>
+                                        </td>
+                                        <td class="align-middle">
+                                            <%- include('../partials/tags', {tags: question.tags}); %>
+                                        </td>
+                                        <% if (locals.course_instance) { %>
+                                            <td class="align-middle">
+                                                <%- include('../partials/assessments', {assessments: question.assessments}); %>
+                                            </td>
+                                            <% } %>
+                                                <td class="align-middle">
+                                                    <%= question.display_type %>
```

Comment:
Can we add a `partial` to render `display_type` with a colored badge? It would be nice to see v2/v3 as different colors.